src/DefaultBuilder/src/StaticWebAssets/StaticWebAssetsHelper.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Xml.Linq;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.FileProviders;
+
+namespace Microsoft.AspNetCore
+{
+    internal static class StaticWebAssetsHelper
+    {
+        private const string StaticWebAssetsManifestName = "Microsoft.AspNetCore.StaticWebAssets.xml";
+        private const string ManifestRootElementName = "StaticWebAssets";
+        private const string VersionAttributeName = "Version";
+        private const string ContentRootElementName = "ContentRoot";
+
+        internal static void UseStaticWebAssets(this IWebHostEnvironment environment)
```

Comment:
re: naming. I would **not** expected to find extension methods on a top-level type like this in this randomly named class.

Reply:
done